src/core/pyproxy.js

```diff
@@ -0,0 +1,565 @@
+// This file to be included from pyproxy.c
+//
+// The point is to make a file that works with JsDoc. JsDoc will give up if it
+// fails to parse the file as javascript. Thus, it's key that this file should
+// parse as valid javascript. `TEMP_EMJS_HELPER` is a specially designed macro
+// to allow us to do this. We need TEMP_EMJS_HELPER to parse like a javascript
+// function call. The easiest way to get it to parse is to make the "argument"
+// look like a function call, which we do with `()=>{`. However, `()=>{` is an
+// invalid C string so the macro needs to remove it. We put `()=>{0,`,
+// TEMP_EMJS_HELPER removes everything up to the comma and replace it with a
+// single open brace.
+// 
+// See definition of TEMP_EMJS_HELPER:
+// #define TEMP_EMJS_HELPER(a, args...)                                           \
+//   EM_JS(int, pyproxy_init, (), UNPAIRED_OPEN_BRACE { args return 0; })
+
+// clang-format off
```

Comment:
So we disable clang-format for all of the file? Was the formatting really not good?

Reply:
It seems that we had disabled clang-format for all ~600 lines of `pyproxy_init_js` in `pyproxy.c` before this PR:
https://github.com/iodide-project/pyodide/blob/f2d6137673e0040e69b05b69a8fd0a0f8c98bd7f/src/core/pyproxy.c#L690

I just checked what happens with the linter turned on and it's... not bad at all. Thanks for pointing that out.